src/ifElse.test.js

```diff
@@ -1,11 +1,45 @@
 'use strict';
 
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
+  const { ifElse } = require('./ifElse');
 
-  it('should ', () => {
+  let firstCallback;
+  let secondCallback;
 
+  beforeEach(() => {
+    firstCallback = jest.fn();
+    secondCallback = jest.fn();
   });
 
   // write tests here
+  it(`should be declared`, () => {
+    expect(ifElse)
+      .toBeInstanceOf(Function);
+  });
+
+  it('should return undefined', () => {
+    const condition = () => true;
+
+    expect(ifElse(condition, firstCallback, secondCallback)).toBeUndefined();
+  });
+
```

Comment:
The test description should be 'should call the first callback...' instead of 'should called the first callback...'. Make sure your descriptions are clear, concise and grammatically correct.